mxcube3/routes/signals.py

```diff
@@ -121,6 +121,15 @@ def sc_unload(location):
 
     socketio.emit('sc', msg, namespace='/hwr')
 
+def path_safe_changed(*args):
+    # responds to the 'pathSafeChanged' signal in cats like SC
```

Comment:
Maybe remove cats specific comment, since this applies basically all sample changers using an robotic arm ?